extension/src/cli/git/index.ts

```diff
@@ -10,6 +10,9 @@ export class GitCli extends Cli {
       cwd
     })
 
-    return standardizePath(await this.executeProcess(options))
+    try {
+      const path = await this.executeProcess(options)
+      return standardizePath(path)
+    } catch {}
```

Comment:
[F] Need to be mindful of calling either CLI in Setup. We don't want things to crash before the users gets a chance to get setup.